File: app/src/main/kotlin/br/com/victorwads/goldenunicorn/data/models/Bank.kt
package br.com.victorwads.goldenunicorn.data.models

import com.google.firebase.firestore.DocumentId
import com.google.firebase.firestore.PropertyName

data class Bank(
    @DocumentId val id: String? = null,
    @PropertyName("name") val name: String = "",
    @PropertyName("fullName") val fullName: String = "",
    @PropertyName("logoUrl") val logoUrl: String = ""
)

File: iOS/Sources/Data/models/Bank.swift
import FirebaseFirestoreSwift

struct Bank: Codable, Identifiable {
    @DocumentID var id: String? = nil
    var name: String
    var fullName: String = ""
    var logoUrl: String

    enum CodingKeys: String, CodingKey {
        case id
        case name
        case fullName
        case logoUrl
    }
}
